boards/arm/sam4l_ek/doc/index.rst

```diff
@@ -0,0 +1,165 @@
+.. _sam4l_ek:
+
+SAM4L-EK
+########
+
+Overview
+********
+
+The SAM4L series embeds picoPower technology for ultra-low power consumption.
+Combined power control techniques are used to bring active current consumption
+down to 90μA/MHz.  The device allows a wide range of configurations giving the
+user the ability to balance between the lowest possible power consumption and
+the feature set selected for the application.  The WAIT and RETENTION modes
+provide full logic and RAM retention, associated with fast wake-up capability
+(<1.5μs) and a very low consumption of, respectively, 3 μA and 1.5 μA.  In
+addition, WAIT mode supports SleepWalking features.  In BACKUP mode, CPU,
+peripherals and RAM are powered off consuming less than 0.9μA with external
+interrupt wake-up support.
+
+The SAM4L-EK is a full featured design to develop for Atmel SAM4L SoC series.
+The kit is equipped with a rich set of peripherals that make the ATSAM4L-EK a
+perfect evaluation platform.  Download the `SAM4L-EK Online User Guide`_ for
+more details.
+
+.. image:: img/atmel-sam4l-ek-callouts.jpeg
+     :width: 500px
+     :align: center
+     :alt: SAM4L-EK
+
+Hardware
+********
+
+- ATSAM4LC4C ARM Cortex-M4 Processor
+- 12 MHz crystal oscillator
+- 32.768 kHz crystal oscillator
+- 1 Micro-AB USB OTG host/device
+- 1 AT86RF2xx IEEE 802.15.4 transceiver connector
+- 1 RS-485 full duplex interface
+- 1 Sensor Xplained board connector
+- 1 Audio Jack connector 3.5mm
+- 1 Dedicated Board Monitor MCU
+
+  - Power measurement (VDDIN, VDDIO, VDDANA)
+  - 1 OLED Display (128x64)
+  - 5 LEDs
+  - 1 Joystick
+  - 1 USART
+  - 1 TWI
+- 1 40x4 LCD Segment Display
+- 1 user touch button and One user pushbutton
+- 1 user LED
+- 1 QTouch Slider
+- 1 QTouch Button
+- 1 TEMT6000 Light Sensor
+- 1 AT25DF641A Serial NOR Flash
+
+Supported Features
+==================
+
+The sam4l_ek board configuration supports the following hardware features:
+
++-----------+------------+-------------------------------------+
+| Interface | Controller | Driver/Component                    |
++===========+============+=====================================+
+| NVIC      | on-chip    | nested vector interrupt controller  |
++-----------+------------+-------------------------------------+
+| SYSTICK   | on-chip    | systick                             |
```

Comment:
Can you add here, MPU if it is present?

Reply:
Update table to show on alphabetic order and added missing MPU and HWINFO lines.